terraform/modules/emr/templates/emr/setup.sh

```diff
@@ -42,34 +42,34 @@ for GROUP in $${COGNITO_GROUPS[@]}; do
   sudo groupadd -f "$GROUP"
 
   echo "Adding users for group $GROUP"
-  USERS=$(aws cognito-idp list-users-in-group --user-pool-id "${user_pool_id}" --group-name "$GROUP" | jq '.Users' | jq -r '.[].Username')
+  USERS=$(aws cognito-idp list-users-in-group --user-pool-id "${user_pool_id}" --group-name "$GROUP" | jq '.Users[]' | jq -r '(.Attributes[] | if .Name =="preferred_username" then .Value else empty end) // .[].Username')
 
   USERDIR=$(aws cognito-idp list-users --user-pool-id "${user_pool_id}")
 
   for USER in $${USERS[@]}; do
```

Comment:
I still don't know why these work on the EMR when they dont work locally in my mac bash. Maybe its a v4 thing.